packages/dialect-react-ui/components/Broadcast/BroadcastForm.tsx

```diff
@@ -31,6 +32,10 @@ function BroadcastForm({ dapp }: BroadcastFormProps) {
   const [isSending, setIsSending] = useState(false);
   const [title, setTitle] = useState('');
   const [message, setMessage] = useState('');
+  const titleLength = useMemo(
+    () => new TextEncoder().encode(title).length,
```

Comment:
minor, but maybe we shouldn't create a new text encoder instance each time?

Reply:
Good point, will update